cli_reference/admin_cli_operations.adoc

```diff
@@ -17,6 +17,12 @@ syntax. You must xref:get_started_cli.adoc#cli-reference-get-started-cli[setup a
 you can perform these operations.
 
 ifdef::openshift-origin,openshift-enterprise[]
+The `oc` command provides command line tools for managing applications, while
+the `oadm` command provides tools for managing a cluster. The `openshift` binary
+is an all-in-one command that can be used for starting services that make up the
+{product-title} cluster. The `openshift` binary can perform all the same actions
+as the `oc` and `oadm` commands.
+
```

Comment:
Since there is already a paragraph talks about `oadm` in https://docs.openshift.org/latest/cli_reference/admin_cli_operations.html#overview , your above paragraph seems to need not mention `oadm` again to avoid duplicating `oadm`.
Since this PR wants to document `openshift`,  you need not mention oc again as duplicated in in get_started_cli.adoc.

How about just to document `openshift` after the `The oadm command ... is used more for project-level operations` **paragraphs** like below?
```
The `openshift` command is used for starting services that make up the {product-title} 
cluster, i.e. `openshift start [master/node]`. But it is also an all-in-one command that 
can perform all the same actions as the `oc` and `oadm` commands via `openshift cli`
 and `openshift admin` respectively
```
This statement not only talks about `openshift` specific command `openshift start`, but also tells 
reader `openshift cli` and `openshift oadm` can use as `oc` and `oadm` when oc/oadm is not available